pages/article_page.rb

```diff
@@ -22,7 +22,7 @@ def self.open(id)
   end
 
 
-  def fill_comment_form(body: nil)
+  def fill_comment_form(body)
```

Comment:
I am not sure it is good idea. I can explain:
- this page is shared for RSpec, Cucumber, Turnip
- in Cucumber we have table for filling out of form. It is easy to convert Cucumber table to hash
- the easiest way to pass hash directly. Just compare
# good
data = CucumberTable # {title: 'foo', body: 'bar'}
fill_comment_form(data)

#bad
fill_comment_form(data[:title], data[:body])